### PR TITLE
Update docker-setup.sh to detect if 'start_arm_container.sh' exists and rename it during docker setup

### LINKED ISSUE #862: 🔧 do not over-write "start_arm_container.sh" if it already exists
### Is this feature missing in the latest version?

- [ ] I'm using the latest release

### Is your feature request related to a problem? Please describe.

When updating to pull down the latest docker image, I didn't realize it would be wiping out my "start_arm_container.sh" script.  It's a minor annoyance to fill it back out, but it would be nice if it detected a file existed there and didn't overwrite it.  Alternatively, if the format of this file is likely to change, I'd love if the install script would rename the old one.

### Describe the solution you'd like?

existing "start_arm_container.sh" script is somehow preserved

### Describe alternatives you've considered?

_No response_

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by TallPatrick**
Added rudimentary handling for checking if "start_arm_container.sh" already exists, and changing the name to "start_arm_container_OLD.sh" if it does, so the new "start_arm_container.sh" can be written

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
List any dependencies that are required for this change.

Fixes #862 

## Type of change
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested locally by running the new code snippet.

- [x] Docker
- [ ] Other (Please state here)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

# Changelog:

Include the details of changes made here
- Single check in, detailed above.

# Logs
N/A

### CODE REVIEW

**Comment by microtechno9000** [comment]
if there is a script in the folder, why download, you could skip the download altogether

File: scripts/docker-setup.sh [hunk 1]
```diff
@@ -80,6 +80,11 @@ function setup_mountpoints() {
 function save_start_command() {
     url="https://raw.githubusercontent.com/automatic-ripping-machine/automatic-ripping-machine/main/scripts/docker/start_arm_container.sh"
     cd ~arm
+    if [ -e start_arm_container.sh ]
+    then
+        echo -e "'start_arm_container.sh' already exists.  Renaming old file to 'start_arm_container_OLD.sh'"
```
Comment (shitwolfymakes) [inline scripts/docker-setup.sh:85]:
```suggestion
        echo -e "'start_arm_container.sh' already exists. Backing up..."
```

File: scripts/docker-setup.sh [hunk 2]
```diff
@@ -80,6 +80,11 @@ function setup_mountpoints() {
...
+        sudo mv /start_arm_container.sh ./start_arm_container_OLD.sh
```
Comment (shitwolfymakes) [inline scripts/docker-setup.sh:86]:
```suggestion
        sudo mv /start_arm_container.sh ./start_arm_container.sh.bak
```

**Review by shitwolfymakes — CHANGES_REQUESTED** [review]
The requested changes are the commonly accepted way of doing this operation

File: scripts/docker-setup.sh (hunk 2 above)
Comment (TallPatrick) [reply scripts/docker-setup.sh:86]: yeah, this is a better solution.  I'm too used to personal projects where additional jank like this is normal for me.

**Comment by TallPatrick** [comment]
@microtechno9000 
> if there is a script in the folder, why download, you could skip the download altogether

Backing up the existing file is generally the safer option in case there is ever a breaking change with "start_arm_container.sh" that is needed all of the values set by users will be retained in the backed-up old version of the script.

**Review by shitwolfymakes — APPROVED** [review]
Great work! Thanks for your help

**Pull request merged**